staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmsizeclient/mockvmsizeclient/interface.go

```diff
@@ -1,4 +1,3 @@
-//go:build !providerless
```

Comment:
Why the go build tag changes here?

Reply:
@andrewsykim this problem has been fixed. PTAL.